pkg/gather/clusterconfig/0_gatherer.go

```diff
@@ -34,7 +36,7 @@ func New(gatherKubeConfig *rest.Config, gatherProtoKubeConfig *rest.Config, metr
 // Gather is hosting and calling all the recording functions
 func (g *Gatherer) Gather(ctx context.Context, recorder record.Interface) error {
 	g.ctx = ctx
-	return record.Collect(ctx, recorder,
+	bulkFns := []func() ([]record.Record, []error){
 		GatherPodDisruptionBudgets(g),
```

Comment:
so the gather functions are closures that return a function to be run to do the actual gathering.

I think this is unnecessary if we do the gathering inside the gatherer and leaving them in just complicates things.
Of course it will complicate the backports a bit but not by terribly.

Thoughts ?

Reply:
LGTM. I think that it would introduce a better readability to the code, removing a necessary layer of complexity.